app/Service/Student/UpdateStudentProfileService.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Service\Student;
+
+use App\Models\Student;
+use App\Exceptions\{
+    StudentNotFoundException,
+    ResumeNotFoundException
+};
+
+class UpdateStudentProfileService
+{
+    public function execute(string $studentId, array $data): bool
+    {
+        return $this->updateStudentProfile($data, $studentId);
+    }
+
+    public function updateStudentProfile(array $data, string $studentId):bool
+    {
+        $student = $this->getStudentById($studentId);
+        $wasStudentUPdated = $this->updateStudent($data, $student);
+        $wasResumeUpdated = $this->updateResume($data, $student);
+
+        return $wasStudentUPdated && $wasResumeUpdated;
+    }
+
+    private function getStudentById(string $studentId): Student
+    {
+        $student = Student::find($studentId);
+        if (!$student) throw new StudentNotFoundException($studentId);
+
+        return $student;
+    }
+
+    private function updateStudent(array $data, Student $student):bool
```

Comment:
subject first then the action (in the parameters)

Reply:
done